pkg/karmadactl/join/join.go

```diff
@@ -150,6 +159,12 @@ func (j *CommandJoinOption) AddFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&j.ClusterProvider, "cluster-provider", "", "Provider of the joining cluster. The Karmada scheduler can use this information to spread workloads across providers for higher availability.")
 	flags.StringVar(&j.ClusterRegion, "cluster-region", "", "The region of the joining cluster. The Karmada scheduler can use this information to spread workloads across regions for higher availability.")
 	flags.StringSliceVar(&j.ClusterZones, "cluster-zones", nil, "The zones of the joining cluster. The Karmada scheduler can use this information to spread workloads across zones for higher availability.")
+	flags.StringVar(&j.KarmadaAs, "karmada-as", "",
+		"Host cluster username to impersonate for the operation. User could be a regular user or a service account in a namespace.")
```

Comment:
What does `Host cluster` refer to?

Reply:
the cluster that the given cluster will join to, "host" is supposed to match the "ClusterConfig.Host" etc naming

Reply:
thanks for your explain.